test/test-maker-taxo.rb

```diff
@@ -20,7 +20,7 @@ def setup
           :link => "http://meerkat.oreillynet.com/?c=cat23",
           :title => "Data: XML",
           :description => "A Meerkat channel",
-         },
+        },
```

Comment:
```suggestion
         },
```

Not in `Style/StringLiterals`.